.cascade-code/Chapman.edu/_cascade/formats/modular/widgets/Shared_Content-Dept_Contacts_Left_Column.vtl

```diff
@@ -0,0 +1,216 @@
+        </symbol>
+    #elseif ($account == "blog")
+        <symbol id="icon-$account" viewBox="0 0 16 16">
+            <title>$account</title>
+            <path d="M2.13 11.733c-1.175 0-2.13 0.958-2.13 2.126 0 1.174 0.955 2.122 2.13 2.122 1.179 0 2.133-0.948 2.133-2.122-0-1.168-0.954-2.126-2.133-2.126zM0.002 5.436v3.067c1.997 0 3.874 0.781 5.288 2.196 1.412 1.411 2.192 3.297 2.192 5.302h3.080c-0-5.825-4.739-10.564-10.56-10.564zM0.006 0v3.068c7.122 0 12.918 5.802 12.918 12.932h3.076c0-8.82-7.176-16-15.994-16z"></path>
+        </symbol>
+    #end
+#end
+## end socialLink macro
+
+## main function within this widget:
+#macro(displayContactsLeftCol $element)  
+##  
+  #set ($widgetXPath = 'contact')
+  #set ($widgetXPath = ${_EscapeTool.xml($widgetXPath)})
+
+    ##set($display =  $_XPathTool.selectSingleNode($element, "contact/display").value)
+    #set($contact = $_XPathTool.selectSingleNode($element, "contact/info/content/system-data-structure"))
+    
+    #set ($display = $element.getChild($widgetXPath).getChild('display').value)
+    #bgColorChooser($widgetXPath)
+    
+    ##fields
+    #set($dept = $_EscapeTool.xml($contact.getChild('dept').value))
+    #set($phone = $_EscapeTool.xml($contact.getChild('phone').value))
+    #set($sPhone = $phone.replace("(", "").replace(")", "").replace("-", "").replace(" ", ""))
+    #set($email = $_EscapeTool.xml($contact.getChild('email').value)) 
+
+    #set($address = $contact.getChild('address'))
+    #set($building = $_EscapeTool.xml($address.getChild('building').value))
+    #set($line = $_EscapeTool.xml($address.getChild('line').value))
+    #set($line2 = $_EscapeTool.xml($address.getChild('line2').value))
+    #set($city = $_EscapeTool.xml($address.getChild('city').value))
+    #set($state = $_EscapeTool.xml($address.getChild('state').value))
+    #set($country = $_EscapeTool.xml($address.getChild('country').value))
+    #set($zip = $_EscapeTool.xml($address.getChild('zip').value))
+    #set($map = $_EscapeTool.xml($address.getChild('map').value))
+    #set($hours = $_SerializerTool.serialize($contact.getChild('hours'), true)) 
+    
+    <div class="callout-box-widget--left callout callout-box-widget--contact-widget #escape($widgetXPath) $bgColor">
```

Comment:
If possible, would be nice to stick to this naming scheme 
`department-contact-widget` followed by `--modifier` like:

`department-contact-widget department-contact-widget--left department-contact-widget--red`

if the Widget's XML Path is camel case and contains 'widget', the following will take care of all of that 
`<div class="${_EscapeTool.xml($widgetXPath)} {_EscapeTool.xml($widgetXPath)}--left $bgColor">`

and will inherit from the wildcard `-widget` classes in  defaults.scss: https://github.com/chapmanu/cascade-assets/blob/f144e832706ce8d6166834d8be5cf9eed0ea8aba/app/assets/stylesheets/base/defaults.scss#L692

Reply:
Cleaned up naming scheme in the existing widget. Chose to leave widgetXpath as 'contact' so we wouldn't have to change the data definition too much. otherwise we would need to scan for pages using the existing widget and update all of them.